src/08_comprehensions.py

```diff
@@ -11,23 +11,26 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
```

Comment:
```suggestion
y = [x for x in range(6)]
```
they're asking for list comprehensions here